modules/apps/object/object-rest-test/src/testFunctional/macros/CustomObjectAPI.macro

```diff
@@ -1,14 +1,126 @@
 definition {
 
+	var headers = '''
+			-u test@liferay.com:test \
+			-H Content-Type: application/json \
+		''';
+
+	macro _curlNObjectEntriesBody {
```

Comment:
this macro doesn't contain curl, only the body right?

Reply:
Yes, I'm not very sure what the suitable name here since it includes both boday for create/POST and update/PUT :thinking: